modules/plan.nix

```diff
@@ -93,6 +93,36 @@ let
             type = bool;
             default = (def.doHaddock or true);
           };
+          doHoogle = mkOption {
+            description = "Also build a hoogle index.";
+            type = bool;
+            default = (def.doHoogle or true);
+          };
+          doHyperlinkSource = mkOption {
+            description = "Link documentation to the source code.";
+            type = bool;
+            default = (def.doHoogle or true);
+          };
+          dontPatchELF = mkOption {
+            description = "If set, the patchelf command is not used to remove unnecessary RPATH entries. Only applies to Linux.";
+            type = bool;
+            default = (def.dontPatchELF or true);
+          };
+          dontStrip = mkOption {
+            description = "If set, libraries and executables are not stripped.";
+            type = bool;
+            default = (def.dontStrip or true);
```

Comment:
I kept this and `dontPatchELF` as defaulting to `true` because that's what they were originally. My personal preference would be to default them to `false`, since that's more often what I want and is the nixpkgs `mkDerivation` default. I found these defaults very surprising.

Reply:
The annoying part with stripping is that it sometimes breaks haskell code. This is mostly due to how haskell code is layed out in object files, and the stripper being a tad bit too agressive.